.github/ServiceX/python/transformer.py

```diff
@@ -156,11 +156,12 @@ def compile_code():
     # Have to use bash as the file runner.sh does not execute properly, despite its 'x'
     # bit set. This seems to be some vagary of a ConfigMap from k8, which is how we usually get
     # this file.
-    r = os.system('bash /generated/runner.sh -c | tee log.txt')
-    if r != 0:
-        with open('log.txt', 'r') as f:
-            errors = f.read()
-            raise RuntimeError("Unable to compile the code - error return: " + str(r)+ 'errors: \n' + errors)
+    # r = os.system('bash /generated/runner.sh -c | tee log.txt')
```

Comment:
is this commented-out code needed? if not, please delete

Reply:
It may be in the future. Marc wanted me to comment it out, but didn't explain further. It's in the xAOD transformer, so I assume it's needed on some level.